libnestutil/logging_event.cpp

```diff
@@ -63,6 +63,9 @@ std::ostream& operator<<( std::ostream& out, const LoggingEvent& e )
   case M_INFO:
     out << "[INFO] ";
     break;
+  case M_DEPRECATED:
+    out << "[INFO] ";
```

Comment:
Wouldn't it be even more expressive if "[INFO]" was "[DEPRECATED]"? This would also allow to search the output log for such functions more easily.